tests/composite_pk/test_checks.py

```diff
@@ -226,7 +226,7 @@ class Foo(models.Model):
         "supports_virtual_generated_columns",
         "supports_stored_generated_columns",
     )
-    def test_composite_pk_cannot_include_generated_field(self):
+    def test_composite_pk_can_include_generated_field(self):
```

Comment:
I'd recommend leaving the test out as the code isn't actively adding generated fields meaning this test is testing a passive feature which _should_ already be tested elsewhere. However there is a cost in keeping a test in place albeit in this case a minor one 🤷 

Ultimately though it's up to you and doesn't bother me either way, I'm mainly stating this if someone asks me why the test is here and see me on blame or something 😅

Reply:
> This tests that generated fields specifically are not disallowed.

the same can be said for any other field type tho, that was the point I meant :) ie we don't test that we're allowing char fields.

>  someone tried to add this back without this test, all tests would pass

this is a fair reason.  I like to just just make sure that folks don't make changes for dogmatic reasons.

this might be a nice thing to add to the contributing documentation.

Reply:
Thank you for the valuable points. I agree in general but given that there was a check and no longer there is one, keeping the test to record that intention adds value, particularly for future code explorers. My opinion is to keep the test.

Reply:
As we now have ticket-36472, I think once that is resolved it might makes sense for us to have a test with a GeneratedField in a CompositePrimaryKey to show there's no recursion error